python/hidet/graph/ops/normalize/norm.py

```diff
@@ -104,12 +105,6 @@ def norm_compute(*indices):
             attributes={'dims': dims, 'accumulate_dtype': accumulate_dtype, 'epsilon': epsilon},
         )
 
-    def allow_prologue(self) -> bool:
-        return False
-
-    def allow_epilogue(self) -> bool:
-        return True
-
```

Comment:
If the cpu and cuda task have different behavior, it is better to create a subclass of the task and override the subclass:
```
class CPUNormalizeTask(NormalizeTask):
    ...
```

Reply:
And implement the resolve rule to convert the normalize operator to corresponding cpu_normalize operator.